docs/resources/mysql_users.md

```diff
@@ -72,19 +72,19 @@ In addition to all arguments above, the following attributes are exported
 
 ### `terraform import` command
 
-* MySQL User can be imported using the `id`. For example:
+* MySQL User can be imported using the `id`:`name`:`name`:... . For example:
 
 ```console
-$ terraform import ncloud_mysql_users.rsc_name 12345
+$ terraform import ncloud_mysql_users.rsc_name 12345:name1:name2
 ```
 
 ### `import` block
 
-* In Terraform v1.5.0 and later, use an [`import` block](https://developer.hashicorp.com/terraform/language/import) to import MySQL User using the `id`. For example:
+* In Terraform v1.5.0 and later, use an [`import` block](https://developer.hashicorp.com/terraform/language/import) to import MySQL User using the `id`:`name`:`name`:... . For example:
 
 ```terraform
 import {
     to = ncloud_mysql_users.rsc_name
-    id = "12345"
+    id = "12345:name1:name2"
```

Comment:
How about changing it not to specify names using mysql_user`s` resource?
For instance, mysql_user`s` resource only accepts the instance id and then imports all of users.
And if some users want to import a single user or part of users, then they should use `mysql_user` resource instead of `mysql_users` (although it does not exist at this moment)

Reply:
When creating a mysql resource, mysql_user is always created as well. Because of this, we are required to specify a name to be managed by the resource.